src/Components/CreateImageWizard/steps/googleCloud.js

```diff
@@ -135,7 +144,7 @@ export default {
       name: 'google-email',
       'data-testid': 'input-google-email',
       type: 'text',
-      label: 'Principal (e.g. e-mail address)',
+      label: 'Email address',
```

Comment:
Please leave the previous "Principal (e.g. e-mail address)" here, this is a newer change the mocks didn't caught up with.

Reply:
np